portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,24 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Alex Najera</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Alex Najera</h1> 
+      <img class="image1" src="/images/meface.png" />
+      
+      <p style="color:grey;">Undergraduate at the University of Texas at Dallas</p>
```

Comment:
We generally prefer to keep all styles in css rather than using inline styles here. So in this case, you can add a class here that you then use to make the color grey in css. See the style guide for more info, https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns.